### PR TITLE
fix(wallet-mobile): Add theme to governance

### PR DESCRIPTION
**Pull request by michalszorad**
## Description / Change(s) / Related issue(s)

Add theme to governance

##### Ticket

[YOMO-1854](https://emurgo.atlassian.net/browse/YOMO-1854)

[YOMO-1854]: https://emurgo.atlassian.net/browse/YOMO-1854?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/ChangeVote/ChangeVoteScreen.tsx [hunk 1]
```diff
@@ -171,20 +173,24 @@ export const ChangeVoteScreen = () => {
...
-    gap: 16,
-  },
-})
+const useStyles = () => {
+  const {color, atoms} = useTheme()
+  const styles = StyleSheet.create({
+    root: {
+      ...atoms.flex_1,
+      ...atoms.justify_between,
+      paddingHorizontal: 18,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/ChangeVote/ChangeVoteScreen.tsx]: @michaeljscript for everything that is not following the standards, from now on, approximate to the closest one
```suggestion
      ...atoms.px_lg,
```

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/ConfirmTx/ConfirmTxScreen.tsx [hunk 2]
```diff
@@ -199,84 +201,84 @@ export const ConfirmTxScreen = () => {
...
+    },
+    totalsArea: {
+      backgroundColor: '#3154cb',
+      ...atoms.py_lg,
+      ...atoms.px_lg,
+      ...atoms.rounded_sm,
+    },
+    total: {
+      fontFamily: 'Rubik-Regular',
+      fontSize: 18,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/ConfirmTx/ConfirmTxScreen.tsx:252]: ping Design, it must have a valid `text_` option.

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/ConfirmTx/ConfirmTxScreen.tsx [hunk 3]
```diff
@@ -199,84 +201,84 @@ export const ConfirmTxScreen = () => {
...
+      lineHeight: 26,
+      color: color.white_static,
+    },
+    totalValue: {
+      fontFamily: 'Rubik-Medium',
+      fontSize: 18,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/ConfirmTx/ConfirmTxScreen.tsx:260]: same

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx [hunk 4]
```diff
@@ -388,19 +389,20 @@ const useStyles = () => {
       fontSize: 20,
       lineHeight: 30,
       color: color.gray_cmax,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx]: `color.text...`

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx [hunk 5]
```diff
@@ -369,14 +369,15 @@ const isTxConfirmed = (txId: string, txInfos: Record<string, TransactionInfo>) =
...
-      flex: 1,
-      justifyContent: 'center',
-      alignItems: 'center',
+      ...atoms.align_center,
+      ...atoms.justify_center,
+      ...atoms.flex_1,
+      backgroundColor: color.bg_color_high,
     },
     button: {
       paddingHorizontal: 24,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx]: I think is `xl`

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx [hunk 6]
```diff
@@ -388,19 +389,20 @@ const useStyles = () => {
...
     supportDescription: {
       fontFamily: 'Rubik-Regular',
       fontSize: 14,
       lineHeight: 22,
       color: color.gray_c600,
-      textAlign: 'center',
+      ...atoms.text_center,
     },
     root: {
       paddingHorizontal: 18,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx]: approximate

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx [hunk 7]
```diff
@@ -388,19 +389,20 @@ const useStyles = () => {
       fontSize: 20,
       lineHeight: 30,
       color: color.gray_cmax,
-      textAlign: 'center',
+      ...atoms.text_center,
     },
     supportDescription: {
       fontFamily: 'Rubik-Regular',
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx]: should have a equivalent `text...`

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx [hunk 8]
```diff
@@ -388,19 +389,20 @@ const useStyles = () => {
...
       fontSize: 14,
       lineHeight: 22,
       color: color.gray_c600,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx]: `text_color...`

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx [hunk 9]
```diff
@@ -409,25 +411,21 @@ const useStyles = () => {
       color: color.gray_c900,
     },
     bold: {
+      ...atoms.font_semibold,
       fontFamily: 'Rubik-Medium',
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/Home/HomeScreen.tsx]: text.._normal I guess

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/WithdrawWarningModal/WithdrawWarningModal.tsx [hunk 10]
```diff
@@ -29,19 +29,19 @@ export const WithdrawWarningModal = ({onParticipatePress}: Props) => {
...
+  const {atoms, color} = useTheme()
   const styles = StyleSheet.create({
     root: {
-      flex: 1,
+      ...atoms.flex_1,
       ...atoms.px_lg,
     },
     text: {
-      color: '#242838',
+      color: color.gray_c900,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/WithdrawWarningModal/WithdrawWarningModal.tsx]: must be text_...

File: apps/wallet-mobile/src/features/Staking/Governance/useCases/WithdrawWarningModal/WithdrawWarningModal.tsx [hunk 11]
```diff
@@ -29,19 +29,19 @@ export const WithdrawWarningModal = ({onParticipatePress}: Props) => {
...
       textAlign: 'center',
       fontFamily: 'Rubik-Regular',
       fontSize: 16,
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Staking/Governance/useCases/WithdrawWarningModal/WithdrawWarningModal.tsx]: should have an equivalent

**Review by stackchain — APPROVED** [review]
looking good, just about the text, font + colors.

**Comment by michalszorad** [comment]
Waiting for a product confirmation

**Comment by stackchain** [comment]
Bringing these ones in.

**Pull request merged**